authmanager/authmanager.go

```diff
@@ -54,7 +54,9 @@ func acquireToken(authCfg *config.Auth, tokenKey string, instructions string) er
 		authCfg.Token = store.Tokens[tokenKey]
 		return nil
 	} else if envToken := os.Getenv(authTokenEnvVariable); envToken != "" {
-		return setAndPersistToken(authCfg, store, tokenKey, envToken)
+		authCfg.Token = envToken
+		store.Tokens[tokenKey] = authCfg.Token
```

Comment:
You don't need the second line. It will store the token in the tokens cache representation in-memory without persisting it to disk (due to not calling `store.Save`.